runtime/fundamentals/web_standards.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Web Standards and Node APIs"
+---
+
+Deno aims to bridge the gap between the browser and the backend with a set of
+web standard APIs that are familiar to web developers. These APIs are designed
+to be compatible with the web platform, making it easier to write code that can
+run in both the browser and Deno.
+
+Existing Node.js ecosystem compatibility is provided through a set of Node.js
+compatible APIs and Deno namespace APIs. These APIs are designed to be
+compatible with Node.js, making it easier to port existing Node.js code to Deno.
+
+Deno also provides its global namespace APIS, to empower developers with
+low-level capabilities, often essential for system-level tasks.
+
+## Web Platform APIs
+
+If you’ve ever built for the browser, you’re already on familiar ground. Deno
+supports standard web APIs (like fetch, WebSockets, and more) instead of
+proprietary alternatives. This means your existing web knowledge can be used in
+your Deno projects.
+
+For more information about the web platform APIs, see the
+[Web Platform APIs](/runtime/reference/web_platform_apis/) documentation.
```

Comment:
this would be a good place to discuss spec deviations and other content contained in https://docs.deno.com/runtime/reference/web_platform_apis/